index.md

```diff
@@ -319,17 +321,21 @@ For more information on processing tags – including the naming and registratio
 
 ### Source
 
-The **source** is optional and starts with a `(':', 0x3A)` character (which is stripped from the value), and if there are no tags it MUST be the first character of the message itself.
+      source          ::=  servername / ( nickname [ [ "!" user ] "@" host ] )
```

Comment:
```suggestion
      source          ::=  servername / ( nickname [ "!" user ] [ "@" host ] )
```

RFC 2812 broke compatibility with RFC 1459 here for no apparent reason.

Reply:
Right.

I think I'll send a PR in the future to say servers SHOULD send user and host, because they rarely do that, it's very annoying for clients, and the only use case I have ever seen is `draft/relaymsg`.